File: apps/worker/firmcode_worker/health.py
from dataclasses import dataclass


@dataclass(frozen=True)
class WorkerHealth:
    service: str
    status: str
    version: str


def get_worker_health(version: str = "0.1.0") -> WorkerHealth:
    return WorkerHealth(service="worker", status="ok", version=version)

File: apps/web/packages/shared/src/contracts/health.ts
export interface HealthResponse {
  service: "api" | "web" | "worker";
  status: "ok";
  version: string;
}

export const createHealthResponse = (
  service: HealthResponse["service"],
  version = "0.1.0"
): HealthResponse => ({
  service,
  status: "ok",
  version
});
